frontend/src/__mocks__/mockSingleModelRegistry.ts

```diff
@@ -0,0 +1,143 @@
+export const mockSingleModelRegistry = {
```

Comment:
```suggestion
export const mockModelRegistry = {
```
Change the file and function name